doc/manuals/devel/mongoBackend.md

```diff
@@ -186,6 +186,7 @@ Regarding the strategy used in `processContextAttributeVector()` to implement en
 * `toPush`: attributes that need to be added to the entity `attrsName` field in the database (list of attribute names), using the [`$addToSet`](https://docs.mongodb.com/manual/reference/operator/update/addToSet) and [`$each`](https://docs.mongodb.com/manual/reference/operator/update/each) operators.
 * `toPull`: attributes that need to be removed from the `attrsName` field in the database (list of attribute names), using the [`$pullAll` operator](https://docs.mongodb.com/manual/reference/operator/update/pullAll).
 * `locAttr` and `geoJson` are related to modifications in the geolocation information associated to the entity (entity `location` field in the database).
+* `dateExpiration` and `replaceDateExpiration` are related to modifications in the TTL expiration date information associated to a transient entity (entity `expDate` field in the database).

```

Comment:
Should be "dateExpirationInPayload" (maybe the change in the name only touched code...)

Reply:
Fixed in 96cd352